### PR TITLE
responsive refinements to header and nav

### PR DESCRIPTION
**Pull request by julietanna**
@kylefiedler
please excuse the extensive comments -- our efforts to make sense of the media queries!

some questions:
--any tips for getting the highlighting on the subnav items to vertically fill the subnav area? it only fills on items that have a 2-line title such as "Living Documents"
--the indenting in _dropdown.scss is wonky, but when I tried to fix it, the dropdown didn't work correctly at sizes smaller than $mobile-navigation-breakpoint.

other than those questions, please respond with any suggestions for syntax, style and more sassiness.

thx
Juliet

### CODE REVIEW

File: mse/local_static/scss/components/_dropdown.scss [hunk 1]
```diff
@@ -9,33 +8,57 @@
...
+    // when viewport is greater than the breakpoint, position dropdown menu below main menu and hide
+    // if greater than or equal to 691 (690 + 1)
+    @include media($mobile-navigation-breakpoint + em(1)) {
       @include position(absolute, 48px 0 null 0);
       display: none;
     }
+    
+    // when viewport is less than or equal to breakpoint, remove dropdown drop shadow
+    @include media(max-width $mobile-navigation-breakpoint) {
+      box-shadow: none;
```
Comment (kylefiedler) [inline mse/local_static/scss/components/_dropdown.scss]: I would reverse the media query here and put the drop shadow in it.

File: mse/local_static/scss/components/_dropdown.scss [hunk 2]
```diff
@@ -9,33 +8,57 @@
...
-      padding: $base-spacing;
+      line-height: $base-spacing * 0.75;
+
+      // when viewport is less than or equal to breakpoint, style dropdowns
+      @include media(max-width $mobile-navigation-breakpoint) {
+        background-color: $primary-color;
+        color: white;
+        line-height: $small-spacing;
+        padding-left: 2.5em;
+        img {
```
Comment (kylefiedler) [inline mse/local_static/scss/components/_dropdown.scss]: Empty line above.

File: mse/local_static/scss/layouts/_footer.scss [hunk 3]
```diff
@@ -1,7 +1,17 @@
 footer {
   background-color: $dark-blue;
-  color: white;
+  color: darken(white, 20%);
+  font-weight: 200;
   padding: $base-spacing;
+
+  @include media(max-width $max-width) {
```
Comment (kylefiedler) [inline mse/local_static/scss/layouts/_footer.scss]: Your indentation is off here.

File: mse/local_static/scss/layouts/_footer.scss [hunk 4]
```diff
@@ -19,4 +29,9 @@ footer {
 
 .footer-search  {
   @include span-columns(8);
+
+  	a {
```
Comment (kylefiedler) [inline mse/local_static/scss/layouts/_footer.scss]: Your indentation is off here.

File: mse/local_static/scss/layouts/_navigation.scss [hunk 5]
```diff
@@ -42,16 +48,25 @@
   display: inline-block;
   vertical-align: top;
 
-  @include media(max-width 925px) {
+  @include media(max-width 975px) {
     font-size: rem(14);
   }
+  
+  @include media(max-width 875px) {
+    font-size: rem(12);
```
Comment (kylefiedler) [inline mse/local_static/scss/layouts/_navigation.scss]: You use this font size in a couple places. Maybe it should be a variable.

Comment (kylefiedler) [inline mse/local_static/scss/layouts/_navigation.scss]: Also you go between `rem` and `px` for font size. I would pick one and be consistent about it.

File: mse/local_static/scss/layouts/_title-area.scss [hunk 6]
```diff
@@ -1,12 +1,32 @@
...
+  margin-bottom: $base-spacing;
     
-    header  {
-        @include outer-container;
+  header  {
+    @include outer-container;
+    
+    @include media(max-width $max-width) {
+    padding: $base-spacing;
+    font-size: $base-font-size * 0.85;
```
Comment (kylefiedler) [inline mse/local_static/scss/layouts/_title-area.scss]: This is the second time I've seen this. Does it deserve it's own variable?

File: mse/local_static/scss/layouts/_title-area.scss [hunk 7]
```diff
@@ -1,12 +1,32 @@
...
   background-color: $toned-blue;
+  margin-bottom: $base-spacing;
     
-    header  {
-        @include outer-container;
+  header  {
+    @include outer-container;
+    
+    @include media(max-width $max-width) {
+    padding: $base-spacing;
```
Comment (kylefiedler) [inline mse/local_static/scss/layouts/_title-area.scss]: Your indentation is off here.

File: mse/local_static/scss/layouts/_title-area.scss [hunk 8]
```diff
@@ -1,12 +1,32 @@
...
+    font-size: $base-font-size * 0.85;
+    }
+
+    @include media(max-width $mobile-navigation-breakpoint) {
+    font-size: rem(12);
```
Comment (kylefiedler) [inline mse/local_static/scss/layouts/_title-area.scss]: Your indentation is off here. Variable?

File: mse/local_static/scss/components/_articles.scss [hunk 9]
```diff
@@ -23,4 +23,10 @@
     @include span-columns(2 8);
     @include omega(4n);
   }
-}
+
+  a h4 {
```
Comment (kylefiedler) [inline mse/local_static/scss/components/_articles.scss]: This is getting fairly specific. Maybe it deserves a class instead?

Comment (julietanna) [inline mse/local_static/scss/components/_articles.scss]: hi Kyle

not sure what you mean -- what exactly is getting fairly specific?

Comment (kylefiedler) [inline mse/local_static/scss/components/_articles.scss]: The selector is more specific. It has 3 selectors. I try to have the least amount of selectors possible.

Comment (julietanna) [inline mse/local_static/scss/components/_articles.scss]: oh, I get what you mean now! thanks!

J

> On Sep 18, 2015, at 1:24 PM, Kyle Fiedler <notifications@github.com> wrote:
> 
> In mse/local_static/scss/components/_articles.scss <https://github.com/knowyourider/mse20/pull/6#discussion_r39879397>:
> 
> > @@ -23,4 +23,10 @@
> >      @include span-columns(2 8);
> >      @include omega(4n);
> >    }
> > -}
> > +
> > +  a h4 {
> The selector is more specific. It has 3 selectors. I try to have the least amount of selectors possible.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/knowyourider/mse20/pull/6/files#r39879397>.
> 

Juliet Jacobson
Design Director, DigitalGizmo
413-628-9949
juliet@digitalgizmo.com

**Pull request closed without merging**